code/__HELPERS/matrices.dm

```diff
@@ -63,3 +67,109 @@
 //The Y pixel offset of this matrix
 /matrix/proc/get_y_shift()
 	. = f
+
+/datum/color_matrix
```

Comment:
Could you also please document this type a bit more? Explain what its purpose is, and what it's used for